ntc_templates/templates/index

```diff
@@ -339,6 +339,7 @@ cisco_ios_show_policy-map.textfsm, .*, cisco_ios, sh[[ow]] pol[[icy-map]]
 cisco_ios_show_power_used.textfsm, .*, cisco_ios, sh[[ow]] pow[[er]] u[[sed]]
 cisco_ios_show_redundancy.textfsm, .*, cisco_ios, sh[[ow]] redu[[ndancy]]
 cisco_ios_show_snmp_group.textfsm, .*, cisco_ios, sh[[ow]] snm[[p]] g[[roup]]
+cisco_ios_show_vlan_group.textfsm, .*, cisco_ios, sh[[ow]] vlan group
```

Comment:
While the entire `vlan` word is required (since the plural form is also possible), `group` can be shortened to just the letter g.

```suggestion
cisco_ios_show_vlan_group.textfsm, .*, cisco_ios, sh[[ow]] vlan g[[roup]]
```